MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -290,6 +305,10 @@ extension InstructionsCardViewController: UICollectionViewDataSource {
         let distance = firstStep ? distanceRemaining : step.distance
         cell.configure(for: step, distance: distance)
         
+        if UIApplication.shared.userInterfaceLayoutDirection == .rightToLeft {
```

Comment:
As in previous comment, let's try to use property recommended by Apple.

I can also see warning related to incorrectly handling layout constraint:
```
2020-12-21 14:58:39.322972-0800 Example[42832:1629484] the item height must be less than the height of the UICollectionView minus the section insets top and bottom values, minus the content insets top and bottom values.
```

I think decrease of UICollectionView height should fix it (should be tested though):
```swift
public func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
    
    return CGSize(width: cardSize.width, height: cardSize.height - 10)
    // return cardSize
}
```